src/ski.go

```diff
@@ -11,7 +11,7 @@ import (
 func main() {
 	opts := parseOptions()
 	validateArgsCount(&opts)
-	opts.postProcessing()
+	opts.postProcessing() // TODO move to parseOptions
```

Comment:
Sowie es aussieht scripts und Commands werden in " umgeschlossen. Fuer die Bearbeitung werden die " entfernt.